inconsistencies.md

```diff
@@ -0,0 +1,153 @@
+# Solving Cassandra consistency
+
+This plan makes the following assumption:
+
+  - We are not willing to rewrite mailbox/cassandra with event sourcing
+  - We will rather identify projections and how to heal them
+
+# Action plan
+
+(prioritized) this section present the conclusion of this studu which details can be found below.
+
+## Grooming
```

Comment:
Does that mean "we still have to figure out what are performance characteristics of LWT and counters"?

Reply:
More: does the performance cost of LWT have a real impact for us.

Reply:
LWT -> "lightweight transactions" right? (tried to catch up on those^^)